### PR TITLE
HPCC4J-502 Update RDF2HPCC deps

### PR DESCRIPTION
**Pull request by rpastrana**
- Updates wsclient dep to cve patched version
- Updates target java version
- Updates wsclient API references

Signed-off-by: Rodrigo Pastrana <rodrigo.pastrana@lexisnexisrisk.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).

## Checklist:
- [x] I have created a corresponding JIRA ticket for this submission
- [x] My code follows the code style of this project.
  - [ ] I have applied the Eclipse code-format template provided.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the HPCC Systems CONTRIBUTORS document (https://github.com/hpcc-systems/HPCC-Platform/wiki/Guide-for-contributors).
- [ ] The change has been fully tested:
  - [ ] This change does not cause any existing JUnits to fail.
  - [ ] I have include JUnit coverage to test this change
  - [ ] I have performed system test and covered possible regressions and side effects.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] This change fixes the problem, not just the symptom

### CODE REVIEW

**Comment by rpastrana** [comment]
megalinter-trivi reported CVEs are resolved

**Comment by rpastrana** [comment]
@jpmcmu please review.

File: rdf2hpcc/src/main/java/org/hpccsystems/rdf/rdf2hpcc/RDFHPCCWsClient.java [hunk 1]
```diff
@@ -440,31 +441,22 @@ public String getRDFStats()
...
         	}
         	
-            HPCCECLDirectClient declient = getEclDirectClient();
-            WorkunitInfo wu = new WorkunitInfo();
+            HPCCWsWorkUnitsClient wuClient = getWsWorkunitsClient();
+            WorkunitWrapper wu = new WorkunitWrapper();
             wu.setECL(eclstats + "\n output(RdfTypeStats('~" + targetHPCCFilePath + "'));");
             wu.setCluster(targetECLCluster);
-            wu.setResultLimit(HPCCECLDirectClient.noresultlimit);
+            wu.setResultLimit(100);
```
Comment (jpmcmu) [inline rdf2hpcc/src/main/java/org/hpccsystems/rdf/rdf2hpcc/RDFHPCCWsClient.java:456]: Any reason for 100 being the result limit here?

**Review by jpmcmu — APPROVED** [review]
@rpastrana looks good

File: rdf2hpcc/src/main/java/org/hpccsystems/rdf/rdf2hpcc/RDFHPCCWsClient.java (hunk 1 above)
Comment (rpastrana) [reply rdf2hpcc/src/main/java/org/hpccsystems/rdf/rdf2hpcc/RDFHPCCWsClient.java:456]: I have to think about that one, we HPCCEclDirect is no longer available, which takes away the ability to set "noresultlimit", if allowed, I will not set the result limit.

**Comment by rpastrana** [comment]
@jpmcmu please review latest change.

**Review by jpmcmu — APPROVED** [review]
@rpastrana looks good

**Pull request merged**